tests/unit/multi_qoi_test_base.py

```diff
@@ -37,6 +37,7 @@ def __init__(self, baseline=None, resolution=10):
         resolution : int
             Number of points returned by each call to this DoI
```

Comment:
can you add documentation for the new arg

Reply:
(and check that the actual docs are updated)

Reply:
The documentation exists for the parent class for now. There were some other requests along documentation when it had to do with parent and inherited classes so I'll figure out the best solution during that task.
I will also update the mkdocs on our next release